code/modules/admin/verbs/playsound.dm

```diff
@@ -80,14 +80,32 @@
 				to_chat(src, "<span class='warning'>For youtube-dl shortcuts like ytsearch: please use the appropriate full url from the website.</span>")
 				return
 			var/shell_scrubbed_input = shell_url_scrub(web_sound_input)
-			var/list/output = world.shelleo("[ytdl] --format \"bestaudio\[ext=mp3]/best\[ext=mp4]\[height<=360]/bestaudio\[ext=m4a]/bestaudio\[ext=aac]\" --get-url \"[shell_scrubbed_input]\"")
+			var/list/output = world.shelleo("[ytdl] --format \"bestaudio\[ext=mp3]/best\[ext=mp4]\[height<=360]/bestaudio\[ext=m4a]/bestaudio\[ext=aac]\" -J --no-playlist -- \"[shell_scrubbed_input]\"")

```

Comment:
--dump-single-json instead of -J, this way people don't need to look up the youtube-dl flags to get an idea of what's going on. This is the same reason I didn't just use -g, and used the full --get-url.